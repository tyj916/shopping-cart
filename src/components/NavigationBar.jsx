import { Link } from "react-router-dom";

function NavigationBar() {
  return (
    <nav>
      <ul>
        <li>
          <Link to='/'>Home</Link>
        </li>
        <li>
          <Link to='shop'>Shop</Link>
        </li>
        <li>
          <button type="button">Cart</button>
        </li>
      </ul>
    </nav>
  );
}

export default NavigationBar;