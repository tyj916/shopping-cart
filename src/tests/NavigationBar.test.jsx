import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import NavigationBar from '../components/NavigationBar';

describe("Navigation bar component", () => {
  it("renders navigation bar", () => {
    const { container } = render(<NavigationBar />);
    expect(container).toMatchSnapshot();
  });
});