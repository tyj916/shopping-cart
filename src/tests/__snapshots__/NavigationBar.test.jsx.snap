// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navigation bar component > renders navigation bar 1`] = `<div />`;
